creating_images/custom.adoc

```diff
@@ -10,49 +10,69 @@
 toc::[]
 
 == Overview
-link:../architecture/core_concepts/builds_and_image_streams.html#custom-build[Custom
-build] is designed to fill the gap that was created when everybody jumped into
-creating Docker images. Still there is a requirement to produce individual
-artifacts (packages, jars, wars, installable zips, base images etc.) This is
-where Custom build is the perfect match to fill in that gap. Additionally Custom
-build allows implementing any extended build process for example, CI/CD flow
-that runs unit or integration tests. The limit here is just the imagination of
-the custom builder image author.
-
-To fully utilize the power of Custom build one needs to be understand how to
-create a builder image that will be capable of building desired objects.
+By allowing you to define a specific builder image responsible for the entire
+build process, OpenShift's
+link:../dev_guide/builds.html#custom-strategy-options[Custom build strategy] was
+designed to fill a gap created with the increased popularity of creating Docker
+images. When there is a requirement for a build to still produce individual
+artifacts (packages, JARs, WARs, installable ZIPs, and base images, for
+example), a _Custom builder image_ utilizing the Custom build strategy is the
+perfect match to fill that gap.
 
+The Custom builder image is a plain Docker image embedded with build process
+logic, for example for building RPMs or base Docker images. The
+`openshift/origin-custom-docker-builder` image from
+https://registry.hub.docker.com/u/openshift/origin-custom-docker-builder[DockerHub]
+is used by default.
```

Comment:
Thanks, changing to:

> The
`openshift/origin-custom-docker-builder` image is available on the
https://registry.hub.docker.com/u/openshift/origin-custom-docker-builder[Docker
Hub] as an example implementation of a Customer builder image.

@bparees In the sentence before it, is that meant to describe specifically this example builder, or is that meant to be true of any custom builder someone may create?